scripts/deploy.sh

```diff
@@ -43,10 +43,14 @@ remove_env() {
 trap remove_env EXIT
 
 source activate $CONDA_ENV
-conda install -q -y "twine==${TWINE_VERSION}"
+echo "installing dependencies"
 conda install -c pytorch -y "pytorch==${PYTORCH_VERSION}"
+python -m pip install "twine${TWINE_VERSION}"
+python -m pip install pyOpenSSL --upgrade
+python -m pip install -r requirements.txt
 python -m pip install -r requirements-dev.txt
 python -m pip install .
+python -m pip list
```

Comment:
This is just informational for the person deploying? Would it make sense to have something like

```bash
echo "Does this look OK?"
read ok
[ "$ok" = "y" ] || exit 0
```

Reply:
It is to help debugging. IMO, asking for confirmation is a bit annoying, as long as everything works fine, there is probably no need to check the installed packages.